app/src/main/java/com/alphawallet/app/widget/AmountDisplay.java

```diff
@@ -0,0 +1,46 @@
+package com.alphawallet.app.widget;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.alphawallet.app.R;
+import com.alphawallet.app.util.LocaleUtils;
+
+import java.math.BigInteger;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import javax.annotation.Nullable;
+
+/**
+ * Created by Jenny Jingjing Li on 21/03/2021
+ * */
+
```

Comment:
This should be called 'AmountDisplayWidget' for consistency with the other widget classes